components/bsx/Gallery/Item/ActionList.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div class="buttons">
     <b-tooltip
+      v-for="action in actions"
+      :key="action"
       class="w-100"
-      :active="!isMakeOffersAllowed"
+      :active="action === ShoppingActions.MAKE_OFFER && !isMakeOffersAllowed"
```

Comment:
This tooltip is not only used for 'Make Offer' action. 
Pls check it again.

<img width="856" alt="image" src="https://user-images.githubusercontent.com/31397967/180636335-8ffbaa0e-ae82-415c-bf08-bb961dc0a151.png">

Reply:
Actually if you look at old codes, it is only used for `Make Offer` action.
1. The tooltip component is only active when `isMakeOffersAllowed` prop is `false`, and 
2. The `isMakeOffersAllowed` prop has only one setter, which is related to `Make offer`:
```
this.isMakeOffersDisabled =
        !this.isMakeOffersAllowed || this.minimumOfferAmount > this.balance
```
which is in `AvailableActions.vue`'s `created` function
```
public async created(): Promise<void> {
    onApiConnect(() => {
      const { api } = Connector.getInstance()
      this.minimumOfferAmount = formatBsxBalanceToNumber(
        api?.consts?.marketplace?.minimumOfferAmount?.toString()
      )
      this.isMakeOffersDisabled =
        !this.isMakeOffersAllowed || this.minimumOfferAmount > this.balance

      if (!this.isMakeOffersAllowed) {
        this.tooltipOfferLabel = this.$t('tooltip.makeOfferDisabled')
      } else if (this.minimumOfferAmount > this.balance) {
        this.tooltipOfferLabel = this.$t('tooltip.makeOfferNotEnoughBalance')
      }
    })
  }
```
3. So what else action need to be used?

Reply:
Thank you for your explanation. You're right.

I have seen `action === ShoppingActions.MAKE_OFFER && !isMakeOffersAllowed` twice in this file. Let's wrap it with a get function. And it's good to go.

Reply:
done:)